scripts/us_epa/airpollution/process.py

```diff
@@ -0,0 +1,377 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This Python Script Load the datasets, cleans it
+and generates cleaned CSV, MCF, TMCF file.
+"""
+
+import os
+from sre_constants import IN
+from absl import app, flags
+import pandas as pd
+import numpy as np
+
+FLAGS = flags.FLAGS
+default_input_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                  "input_files")
+flags.DEFINE_string("input_path", default_input_path, "Import Data File's List")
+
+_MCF_TEMPLATE = ("Node: dcid:{pv1}\n"
+                 "typeOf: dcs:StatisticalVariable\n"
+                 "populationType: dcs:Emissions\n"
+                 "emissionSourceType: dcs:NonBiogenicEmissionSource\n"
+                 "measurementQualifier: dcs:Annual{pv2}{pv3}\n"
+                 "statType: dcs:measuredValue\n"
+                 "measuredProperty: dcs:amount\n")
+
+_TMCF_TEMPLATE = (
+    "Node: E:airpollution_emission_trends_tier1->E0\n"
+    "typeOf: dcs:StatVarObservation\n"
+    "variableMeasured: C:airpollution_emission_trends_tier1->SV\n"
+    "measurementMethod: C:airpollution_emission_trends_tier1->"
+    "Measurement_Method\n"
+    "observationAbout: C:airpollution_emission_trends_tier1->geo_ID\n"
+    "observationDate: C:airpollution_emission_trends_tier1->year\n"
+    "scalingFactor: 1000\n"
+    "unit: Ton\n"
+    "value: C:airpollution_emission_trends_tier1->observation\n")
+
+
+def _add_missing_columns(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Aggregates the columns based on SV
+
+    Args: df (pd.DataFrame): df as the input, to aggregate values
+
+    Returns: df (pd.DataFrame): modified df as output
+    """
+
+    df = df.drop(df[(df['SV'].str.contains('PrescribedFire')) |
+                    (df['SV'].str.contains('Wildfire')) |
+                    (df['SV'].str.contains('Miscellaneous'))].index)
```

Comment:
Can you please add comment about functionality here.

Reply:
Done